packages/app/pages/organization/course/add.tsx

```diff
@@ -81,6 +85,14 @@ export default function Add(): ReactElement {
       const courseTimezoneField = formValues.courseTimezone
       const semesterIdField = formValues.semesterId
       const profIds = isAdmin ? formValues.professorsUserId : [profile.id]
+      const courseFeatures = [
+        'chatBotEnabled',
```

Comment:
this appears twice in the class, consider making as enum / shared component

Reply:
It might look like code duplication, but the second one is listing all of the default checked features. I will add a comment for this though, so thank you